Remove WebKitSetIsClassic check

LumioseSil opened the pull request:
#### 85c350cb0b0b9e0f1e8c762caa208a8904273536
<pre>
Remove WebKitSetIsClassic check
<a href="https://bugs.webkit.org/show_bug.cgi?id=252822">https://bugs.webkit.org/show_bug.cgi?id=252822</a>

Reviewed by Alex Christensen.

This function is no longer being called, so we can remove it.

*Source\WebKitLegacy\ios\Misc\WebUIKitSupport.h:
*Source\WebKitLegacy\ios\Misc\WebUIKitSupport.mm:
*Source\WebKitLegacy\ios\WebKit.iOS.exp:

Canonical link: <a href="https://commits.webkit.org/260832@main">https://commits.webkit.org/260832@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1d945b36a12bf570ce647a28cd5e27121c71f598

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109346 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18421 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42147 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/879 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118546 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19970 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9699 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101579 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115102 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14886 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98113 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43068 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96858 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29766 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84819 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11245 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31109 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11920 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8040 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17287 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50712 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7511 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13545 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

achristensen07 reviewed (approved):
I'm approving this after having searched internally.  However, cleanups like this in WebKitLegacy are likely wasted effort.

LumioseSil committed:
Remove WebKitSetIsClassic check
https://bugs.webkit.org/show_bug.cgi?id=252822

Reviewed by Alex Christensen.

This function is no longer being called, so we can remove it.

*Source\WebKitLegacy\ios\Misc\WebUIKitSupport.h:
*Source\WebKitLegacy\ios\Misc\WebUIKitSupport.mm:
*Source\WebKitLegacy\ios\WebKit.iOS.exp:

Canonical link: https://commits.webkit.org/260832@main
